scripts/image_upload_glance.py

```diff
@@ -27,16 +30,17 @@ def parse_cmd_line():
     parser = argparse.ArgumentParser(description=__doc__,
                                      formatter_class=argparse.RawDescriptionHelpFormatter)
 
-    parser.add_argument('--image', help='Image to be uploaded to Glance', required=True)
+    parser.add_argument('--image', help='Image to be uploaded to Glance', required=False)
```

Comment:
REQUIRED: As far as I understand one of parameters --image or --url is required. Please add a check that at least one of them is present.

Reply:
@izapolsk , done